app/src/main/java/com/sopt/now/gs/feature/reserve/ReserveBannerAdapter.kt

```diff
@@ -0,0 +1,25 @@
+package com.sopt.now.gs.feature.reserve
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.sopt.now.gs.R
+import com.sopt.now.gs.databinding.ItemReserveTopBannerBinding
+
+class ViewPagerAdapter(private val listData: ArrayList<ReserveBannerPage>) : RecyclerView.Adapter<ReservePageViewHolder>() {
```

Comment:
```suggestion
class ViewPagerAdapter(private val listData: ArrayList<ReserveBannerEntity>) : RecyclerView.Adapter<ReservePageViewHolder>() {
```
data class는 끝이 Data 혹은  Entity로 끝나는게 좋습니다!